src/Personal/index.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Button } from 'react-native';
+import { useNavigation } from 'react-navigation-hooks';
+import { useSelector } from 'react-redux';
+import RegisterView from './RegisterView';
+import Personal from './personal';
+
+export default function Index() {
```

Comment:
The component should have a more descriptive name, or you can just simply omit the name itself. But `Index` should be avoided